strassengezwitscher/mapobjects/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.conf.urls import url
 from . import views
 
+app_name = 'mapobjects'
```

Comment:
Nope. Hier erklärt: https://docs.djangoproject.com/en/1.9/topics/http/urls/#url-namespaces-and-included-urlconfs

Ich benutze es indirekt in meinen Tests mit `url = reverse('mapobjects:list')`